main.go

```diff
@@ -165,9 +165,18 @@ func executeRulesets(rulesets []windup.Ruleset, datadir string) (string, string,
 			}
 		}
 
-		err = os.WriteFile(filepath.Join(datadir, JAVA_PROJECT_DIR, "pom.xml"), []byte(POM_XML), 0644)
-		if err != nil {
-			return "", "", err
+		_, err = os.Stat(filepath.Join(datadir, "pom.xml"))
```

Comment:
would you mind adding a comment to explain this?

Reply:
done